include/dpp/discordvoiceclient.h

```diff
@@ -50,7 +50,7 @@
 #include <functional>
 #include <chrono>
 
-
+#define SEND_AUDIO_RAW_MAX_LENGTH 11520
```

Comment:
If you want to let users define their own size, then you should wrap this in `#ifndef`. Otherwise, this should be a constexpr variable, not a macro.

Reply:
I don't want users to define their own size, I want users to have access to the size without hardcoding it in their codebase. What's the benefit of constexpr variable than macros here that is should be a constexpr variable?

Reply:
Awesome, thanks for changing it! To answer your question, constexpr variables have a type, which makes them less prone to subtle bugs introduced by like implicit conversions or integer promotions. They're also namespaced, so there is no risk of conflicting with a user's code.